src/Components/ImageOrVideo.tsx

```diff
@@ -31,15 +31,21 @@ export const ImageOrVideo = connect(function ImageOrVideo<T extends string>({
 
   useEffect(() => {
     const mediaQuery = window.matchMedia('(prefers-reduced-motion: reduce)')
-    setShouldAutoplay(!mediaQuery.matches)
-    if (mediaQuery.matches) setIsPaused(true)
-
-    const handleChange = () => {
-      setShouldAutoplay(!mediaQuery.matches)
+    const updatePreferences = () => {
+      if (mediaQuery.matches) {
```

Comment:
* [ ] FYI Naming the `mediaQuery` after what it queries would improve readability. E.g. `if (prefersReducedMotion.matches)`.